ml_service/util/create_scoring_image.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -euo pipefail # strict mode, fail on error
+set -x # verbose
+
+docker run \
```

Comment:
Eugene already has a pipeline file to do this and not the best idea to do docker-in-docker.  I'd prefer you remove this file and we can work into a different PR.

Reply:
This is run as part of the *release pipeline* directly on the agent, so no docker-in-docker.

I agree this script is pretty ugly-looking. It is removed anyway with PR #122.

If I just remove the file, the getting_started file will not have complete instructions to release a webapp...